src/components/tree-view/tree-view.tsx

```diff
@@ -68,7 +74,12 @@ export const TreeItem: React.SFC<TreeItemProps> =
         };
 
         return (
-            <div>
+            <li
+                aria-expanded={item.children ? !!state!.isExpanded : undefined}
```

Comment:
I think you should add a test that makes sure that the correct roles / aria and whatnot is added to the correct elements

Reply:
agreed